core/include/utils/motion_controller.h

```diff
@@ -43,6 +43,17 @@ class MotionController : public Feedback
      * @param ff_cfg Definitions of kS, kV, and kA
      */
     MotionController(m_profile_cfg_t &config);
+    /**
+     * @brief Construct a new Motion Controller object
+     * 
+     * @param max_v Maximum velocity the movement is capable of
```

Comment:
can we make it clear that these parameters are passed through the m_profile_cfg_t struct?

Reply:
yes